src/platform/forms-system/src/js/helpers.js

```diff
@@ -154,9 +154,19 @@ export function hideFormTitle(formConfig, pathName) {
   const pageList = createPageList(formConfig, formPages);
   const page = pageList.find(p => p.path === pathName);
 
-  if (!page || !page.chapterKey) return false;
+  if (pathName === '/confirmation') {
```

Comment:
Should this apply to the `/introduction` page too? Or does it already somehow?

Reply:
Good idea to check to be sure - I looked - Introduction page always has its title already omitted in FormApp logic (it's a little confusing where what owns what 😅), so consumer IntroductionPage code determines it instead, so no worries about any additional change here.